### PR TITLE
Add flag in flattenOCJSON to remove namespace from path

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

File: gnmidiff/json.go [hunk 1]
```diff
@@ -123,7 +127,12 @@ func flattenOCJSONAux(root interface{}, path string, leaves map[string]interface
 	case map[string]interface{}:
 		// This is a container or a list element.
 		for subpath, subv := range v {
-			if err := flattenOCJSONAux(subv, path+"/"+subpath, leaves); err != nil {
+			if !keepNamespace {
+				if idx := strings.IndexRune(subpath, ':'); idx != -1 {
```
Comment (robshakir) [inline gnmidiff/json.go:131]: I'm not sure which is more efficient, but this seemed a bit more logical to me -- completely aesthetic though, so feel free to ignore this.

```golang
pp := strings.Split(subpath, ":")
subpath = pp[len(pp)-1]
```

File: gnmidiff/json.go [hunk 2]
```diff
@@ -55,20 +56,23 @@ import (
 //   "openconfig-network-instance:config/type": "openconfig-network-instance-types:L3VRF",
 //   "openconfig-network-instance:name": "RED",
 // }
-func flattenOCJSON(json7951 []byte) (map[string]interface{}, error) {
-	// TODO: Add option to remove the namespace on paths of returned updates.
+//
+// When keepNamespace=false, then any namespace is removed from the flattened
+// *paths*, but still kept in any enum values.
```
Comment (robshakir) [inline gnmidiff/json.go:61]: (nit) identity values, since enumerations don't have this namespacing.

File: gnmidiff/json.go (hunk 1 above)
Comment (wenovus) [reply gnmidiff/json.go]: Nice one, updated. yes I was aiming for some aesthetics here.

File: gnmidiff/json.go (hunk 2 above)
Comment (wenovus) [reply gnmidiff/json.go]: Done.

**Pull request merged**